src/transactions/feed/detailContent/SwapContent.tsx

```diff
@@ -0,0 +1,132 @@
+
+export interface Props {
+  exchange: TokenExchange
+}
+
+// Note that this is tested from TransactionDetailsScreen.test.tsx
+export default function SwapContent({ exchange }: Props) {
+  const { t } = useTranslation()
+  const tokensList = useSelector(tokensListSelector)
+
+  const fromTokenSymbol = tokensList.find(
+    (token) => token.address === exchange.outAmount.tokenAddress
+  )?.symbol
+  const toTokenSymbol = tokensList.find((token) => token.address === exchange.inAmount.tokenAddress)
+    ?.symbol
+
+  const onPressTxDetails = () => {
+    ValoraAnalytics.track(SwapEvents.swap_feed_detail_view_tx)
+    navigate(Screens.WebViewScreen, {
+      uri: `${networkConfig.celoExplorerBaseTxUrl}${exchange.transactionHash}`,
+    })
+  }
+
+  return (
+    <View style={styles.contentContainer}>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.bodyText}>{t('swapTransactionDetailPage.swapTo')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.outAmount.value}
+          tokenAddress={exchange.outAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          hideSign={true}
+          testID="SwapContent/swapFrom"
+        />
+      </View>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.bodyText}>{t('swapTransactionDetailPage.swapFrom')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.inAmount.value}
+          tokenAddress={exchange.inAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          showExplicitPositiveSign={true}
+          hideSign={true}
+          testID="SwapContent/swapTo"
+        />
+      </View>
+      <View style={styles.separator} />
+      <View style={[styles.row, { paddingBottom: Spacing.Smallest8 }]}>
+        <Text style={styles.bodyText}>{t('swapTransactionDetailPage.rate')}</Text>
+        <Text testID="SwapContent/rate" style={styles.currencyAmountText}>
+          {`1 ${fromTokenSymbol} ≈ ${formatValueToDisplay(
+            new BigNumber(exchange.inAmount.value).dividedBy(exchange.outAmount.value)
+          )} ${toTokenSymbol}`}
+        </Text>
+      </View>
+      <View style={[styles.row, { paddingBottom: Spacing.Smallest8 }]}>
+        <Text style={styles.bodyText}>{t('swapTransactionDetailPage.estimatedFee')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.fees[0].amount.value}
+          tokenAddress={exchange.fees[0].amount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          hideSign={true}
+          testID="SwapContent/estimatedFee"
+        />
+      </View>
+      <Touchable testID={'txDetails'} borderless={true} onPress={onPressTxDetails}>
+        <View style={styles.rowContainer}>
```

Comment:
nit: i think that this style can be applied to the `Touchable` and then this can be replaced with a fragment like `<>`?

Reply:
I did that!